NetSDK/Services/Client/Classes/SplitClient.cs

```diff
@@ -75,5 +76,48 @@ private void RecordStats(string key, string feature, long start, string treatmen
                 treatmentLog.Log(key, feature, treatment, start);
             }
         }
+
+        private void RecordStats(Key key, string feature, long start, string treatment, string operation, Stopwatch clock)
+        {
+            if (metricsLog != null)
+            {
+                metricsLog.Time(SdkGetTreatment, clock.ElapsedMilliseconds);
+            }
+
+            if (treatmentLog != null)
+            {
+                treatmentLog.Log(key, feature, treatment, start);
+            }
+        }
+
+
+        public string GetTreatment(Domain.Key key, string feature, Dictionary<string, object> attributes = null)
```

Comment:
what happens if the `bucketing_key` is null? if that is the case, we should default to the standard get_treatment method.

Can we also add a uni test for that?